desktop/core/requirements.txt

```diff
@@ -10,7 +10,7 @@ configobj==5.0.6
 cryptography==3.2
 django-auth-ldap==1.3.0
 Django==2.0.13 # Django 2 then 3?
-django-axes==2.2.0
+django-axes==4.5.4
```

Comment:
Because of this error `'django.contrib.auth.views' has no attribute 'login'
`

Reply:
Not updated to the latest version because of Deprecate  Python 2.7 and Django 1.11.

Reply:
Requirements.txt is Python 3 only and we should target the latest one. We could go incrementally there too, 4.5, then v5